swan-pipeline/src/main/java/de/fraunhofer/iem/swan/features/code/MethodStartsWithStringFeature.java

```diff
@@ -0,0 +1,77 @@
+package de.fraunhofer.iem.swan.features.code;
+
+import de.fraunhofer.iem.swan.data.Category;
+import de.fraunhofer.iem.swan.data.Method;
+import de.fraunhofer.iem.swan.features.code.type.IFeature;
+import de.fraunhofer.iem.swan.features.code.type.MethodNameStartsWithFeature;
+import de.fraunhofer.iem.swan.features.code.type.WeightedFeature;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import static de.fraunhofer.iem.swan.features.code.SecurityVocabulary.*;
+
+public class MethodStartsWithStringFeature extends WeightedFeature implements IFeatureNew {
```

Comment:
We could include this in the new approach that we discussed. For example, multiply the weights of the word based on the position in the text.